Formula/noweb.rb

```diff
@@ -6,6 +6,11 @@ class Noweb < Formula
   sha256 "c913f26c1edb37e331c747619835b4cade000b54e459bb08f4d38899ab690d82"
   license "Noweb"
 
+  livecheck do
+    url "https://deb.debian.org/debian/pool/main/n/noweb/"
+    regex(/href=.*?noweb[._-]v?(\d+(?:\.\d+)+)\.orig\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?noweb[._-]v?(\d+(?:\.\d+)+[a-z]?)\.orig\.t/i)
```

Almost missed this but the current formula version is `2.11b` (they use `a`, `b`, `c`, and it's not `b` for beta), so we'll need to match an optional letter after the version.